### DISCUSSION TITLE
在NoneBot2的教程中说调用平台API是指什么API？在哪里查看？

### DISCUSSION THREAD

**Discussion by icewizardry** [nonebot/discussions#245] [问答]
我想做一个能主动发送消息的机器人，但是需要用到平台api调用，但是官方的主动调用API说明太简单了，不是很明白，能否给些更多的提示？

**Comment by yanyongyu** [comment]
文档里的是调用api的样例格式，具体api的名称和参数，请看对应平台的文档

**Reply by icewizardry** [reply]
对应平台的文档是指啥？NoneBot2里发送消息为什么需要看其他平台的API？

**Reply by yanyongyu** [reply]
你要用gocq发消息就去看gocq的api，你要用飞书发消息就看飞书api

**Reply by icewizardry** [reply]
原来如此，谢谢

**Reply by icewizardry** [reply]
大佬，我现在跑起来的nb2是和qq交互的，那平台API是去看go-cqhttp的API，go-cqhttp是连接nb2和qq的主要通道，这个我理解对吗？

**Reply by yanyongyu** [reply]
gocq和nb之间是通过onebot v11协议通讯，api实现的列表以gocq为准

**Reply by icewizardry** [reply]
那如果我想接企业微信该怎么操作呢？

**Reply by yanyongyu** [reply]
目前没有企业微信适配器

**Reply by icewizardry** [reply]
非常感谢，大佬辛苦了

**Reply by icewizardry** [reply]
企业微信还是挺重要的，目前你们是否有适配企业微信的计划？

**Reply by yanyongyu** [reply]
人手不足暂无计划，欢迎社区贡献
